src/lib/ngsi10/SubscribeContextRequest.h

```diff
@@ -54,6 +55,7 @@ typedef struct SubscribeContextRequest
   Restriction            restriction;            // Optional
   NotifyConditionVector  notifyConditionVector;  // Optional
   Throttling             throttling;             // Optional
+  MaxFailsLimit          maxFailsLimit;          // Optional
```

Comment:
Why do you need a whole class for this? Just an usigned int should suffice...

Reply:
Morevover, I'd realized this is the NGSIv1 subscription request class.

NGSIv1 is deprecated, so no need of modifying this file in anyway. The new functionality will be only for NGSIv2.